### PR TITLE
Make question tile questions center aligned

### PR DESCRIPTION
**Pull request by AddisonDunn**
## Product Description

Jira: [USH-3941](https://dimagi-dev.atlassian.net/browse/USH-3941?atlOrigin=eyJpIjoiZWRjNDAwNjhjYWM3NGQwMzkzZjc3ZTdiODBmZmUxOTQiLCJwIjoiaiJ9).

Aligns the questions in a question tile row along the center:

Screenshot 2023-12-15 at 2 38 20 PM

[Link to feature docs](https://confluence.dimagi.com/display/commcarepublic/Advanced+Web+Apps+Formatting#AdvancedWebAppsFormatting-QuestionTiles(%22n%22-per-row))

Where as before questions were by default aligned along the top and inconsistently organized:

image

## Technical Summary

This PR just adds a CSS class that makes the row div a center-aligned flexbox and then removes the padding of certain kind of descendants, since some question types add weird padding that messes up the center alignment.

## Feature Flag

None, but this is Web Apps only.

### Safety story

- Will test locally and on staging
- Planning to ask QA to go through BHA-like apps and test out different forms/question types
- Purely UI change

### Automated test coverage

None since its just CSS for now.

### QA Plan

Will be requested.

### Rollback instructions

- [X] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [X] Risk label is set correctly
- [X] The set of people pinged as reviewers is appropriate for the level of risk of the change

[USH-3941]: https://dimagi-dev.atlassian.net/browse/USH-3941?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: corehq/apps/hqwebapp/static/cloudcare/less/formplayer-webapp/form.less [hunk 1]
```diff
@@ -193,3 +193,15 @@
     font-size: 14px;
   }
 }
+
+.question-tile-row {
+  display: flex;
+  align-items: center;
+  margin-bottom: 20px;
```
Comment (MartinRiese) [inline corehq/apps/hqwebapp/static/cloudcare/less/formplayer-webapp/form.less:200]: That is a lot of margin

**Comment by orangejenny** [comment]
QA and delivery should definitely look at this, so we have a good understanding of how things look with different question types and label lengths. I'm a bit concerned that if forms mix short and long labels, alignment will get out of whack.

**Review by Jtang-1 — APPROVED** [review]
tested it around a bit on staging and things look good

**Comment by orangejenny** [comment]
@AddisonDunn QA having tested a referrals prototype app is great. I did run the QA doc by delivery, and they gave a 👍 , so you're good to merge.

**Pull request merged**